packs/familiar-abilities/construct.json

```diff
@@ -0,0 +1,40 @@
+{
+    "_id": "5eqcXtPsYunphMOZ",
+    "img": "systems/pf2e/icons/actions/Passive.webp",
+    "name": "Construct",
+    "system": {
+        "actionType": {
+            "value": "passive"
+        },
+        "actions": {
+            "value": null
+        },
+        "category": "familiar",
+        "description": {
+            "value": "<p style=\"text-align:justify\">Your familiar has the construct trait instead of the animal trait. The familiar is immune to death effects, disease, doomed, drained, fatigued,<br />healing, nonlethal attacks, paralyzed, poison, sickened, spirit, unconscious, vitality, and void. Your familiar must have the @UUID[Compendium.pf2e.familiar-abilities.Item.Tough] pet ability to select this.</p>"
```

Comment:
```suggestion
            "value": "<p>Your familiar has the construct trait instead of the animal trait. The familiar is immune to death effects, disease, doomed, drained, fatigued,<br />healing, nonlethal attacks, paralyzed, poison, sickened, spirit, unconscious, vitality, and void. Your familiar must have the @UUID[Compendium.pf2e.familiar-abilities.Item.Tough] pet ability to select this.</p>"
```